### PR TITLE
add passport NFC reader example

### PR DESCRIPTION
**Pull request by Superkachenya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "react-native-reanimated": "1.8.0",
     "react-native-safe-area-context": "1.0.0",
     "react-native-safe-area-view": "1.1.1",
-    "react-native-scanbot-sdk": "4.5.0",
+    "react-native-scanbot-sdk": "4.5.0-dev6",
```
Comment (MrDyro) [inline package.json:27]: Update to version 4.6.0 after the SDK is published

**Review by MrDyro — CHANGES_REQUESTED** [review]
Update version of the SDK

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "react-native-reanimated": "1.8.0",
     "react-native-safe-area-context": "1.0.0",
     "react-native-safe-area-view": "1.1.1",
-    "react-native-scanbot-sdk": "4.5.0",
+    "react-native-scanbot-sdk": "4.6.0-beta1",
```
Comment (MrDyro) [inline package.json:27]: Change to 4.6.0 after the SDK was published.

**Review by MrDyro — APPROVED** [review]
After the release of SDK v4.6.0 change the dependency here and merge it to master.

**Review by eddyfrank — APPROVED** [review]

**Pull request merged**